src/render/utils/is-draggable.ts

```diff
@@ -0,0 +1,6 @@
+import { VisualElement } from "../types"
+
+export function isDraggable(visualElement: VisualElement) {
+    const { drag, _dragX } = visualElement.getProps()
+    return drag && !_dragX
```

Comment:
What does the _dragX mean?

Reply:
This is the external `_dragX` and `_dragY` prop - if we're outputting drag to those we can consider this component not really draggable

Reply:
(We always use both hence the single check)